installers/msi-language/Product.p.wxs

```diff
@@ -213,11 +222,16 @@
 		<CustomAction
 			Id="SetStateDeployPropertyValues"
 			Property="StateDeploy"
-			Value="INSTALLDIR=[INSTALLDIR];PROJECT_OWNER_AND_NAME=[PROJECT_OWNER_AND_NAME];IS_MODIFY=[IS_MODIFY];STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];STATE_TOOL_PATH=[STATE_TOOL_PATH];AS_USERNAME=[AS_USERNAME];AS_PASSWORD=[AS_PASSWORD];AS_TOTP=[AS_TOTP];UI_LEVEL=[UILevel];MSI_VERSION=[MSI_VERSION];PRODUCT_VERSION=[ProductVersion];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];ERROR=[ERROR];ERROR_MESSAGE=[ERROR_MESSAGE];PRODUCT_NAME=[ProductName];USERSID=[UserSID];COMMIT_ID=[CommitID];INSTALL_MODE=[INSTALL_MODE]"
+			Value="INSTALLDIR=[INSTALLDIR];PROJECT_OWNER_AND_NAME=[PROJECT_OWNER_AND_NAME];IS_MODIFY=[IS_MODIFY];STATE_TOOL_INSTALLED=[STATE_TOOL_INSTALLED];STATE_TOOL_PATH=[STATE_TOOL_PATH];AS_USERNAME=[AS_USERNAME];AS_PASSWORD=[AS_PASSWORD];AS_TOTP=[AS_TOTP];UI_LEVEL=[UILevel];MSI_VERSION=[MSI_VERSION];PRODUCT_VERSION=[ProductVersion];MsiLogFileLocation=[MsiLogFileLocation];PRIVACY_ACCEPTED=[PRIVACY_ACCEPTED];ERROR=[ERROR];ERROR_MESSAGE=[ERROR_MESSAGE];PRODUCT_NAME=[ProductName];USERSID=[UserSID];COMMIT_ID=[CommitID];INSTALL_MODE=[INSTALL_MODE];LAST_DIALOG=[LAST_DIALOG]"
```

Comment:
I don't think it's necessary to pass `LAST_DIALOG` into any of the deferred custom actions, everything you're working with here is in an `immediate` context. (ie. you're not using `session.CustomActionData["LAST_DIALOG"]` anywhere)